lmdeploy/tokenizer.py

```diff
@@ -173,22 +166,20 @@ def _maybe_add_prefix_space(self, tokens, decoded):
         else:
             return decoded
 
-    def encode(self, s: str):
+    def encode(self, s: str, sequence_start: bool = True, **kwargs):
```

Comment:
I suggest using `add_special_token`, decoupling `Tokenizer` and the sequence status

Reply:
`add_special_token` means more than `sequence_start` or `add_bos`. Maybe I should rename the argument `sequence_start` to `add_bos`. `add_bos` is an argument of SentencePieceTokenizer model `Encode`.

Reply:
`add_bos` is fine.